tests/e2e/utils/CheReporter.ts

```diff
@@ -181,12 +181,48 @@ class CheReporter extends mocha.reporters.Spec {
 			const networkLogsEntries: logging.Entry[] = await this.driverHelper.getDriver().manage().logs().get('performance');
 			const events: any[] = networkLogsEntries.map((entry): any[] => JSON.parse(entry.message).message);
 			const har: any = chromeHar.harFromMessages(events, { includeTextFromResponseBody: true });
+			try {
+				this.maskHarContent(har);
+			} catch (e) {
+				console.log(e);
```

Comment:
can be `Logger.err(e)`

Reply:
Oops, the try catch was something I forgot to remove, can we just remove it @nallikaea ?

Reply:
I removed it, but please let me know if you would like to keep it